### PR TITLE
Fix datetime issues

### PR DESCRIPTION
**Pull request by alex0popa**

### CODE REVIEW

**Review by andreisamoila74 — APPROVED** [review]

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]
using 'localstamp' is not solving your problem: https://metacpan.org/pod/Time::Stamp#localstamp -> you can read this

File: codewell-fe/src/Helpers/Utils.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import moment from 'moment-timezone';
+
+export function getDatePosted(utcDate) {
```
Comment (S7012MY) [inline codewell-fe/src/Helpers/Utils.js:3]: convertToLocalTimezone

**Review by S7012MY — CHANGES_REQUESTED** [review]

File: codewell-fe/src/Helpers/Utils.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import moment from 'moment-timezone';
+
+// Convert utc date to local time zone date
```
Comment (andreisamoila74) [inline codewell-fe/src/Helpers/Utils.js:3]:
```suggestion
// Convert UTC date to local time zone date
```

File: codewell-fe/src/Helpers/Utils.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+export function convertToLocalTimezone(utcDate) {
+  let timeZone = moment.tz.guess(true);
+  let datePosted = moment.utc(utcDate).tz(timeZone).format(
+    'YYYY-MM-DD HH:mm:ss');
```
Comment (andreisamoila74) [inline codewell-fe/src/Helpers/Utils.js:7]:
```suggestion
  let datePosted = moment.utc(utcDate).tz(timeZone)
    .format('YYYY-MM-DD HH:mm:ss');
```

File: codewell-fe/src/__tests__/Helpers/Utils.test.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+const utcDate = '2020-01-01 12:00:00';
+
+describe('Test function convertToLocalTimezone()', () => {
+  test('Conversion to timezone Europe/Rome', () => {
+    moment.tz.guess = function () {
+      return  'Europe/Rome';
+    }
+    expect(convertToLocalTimezone(utcDate)).toStrictEqual(
+      '2020-01-01 13:00:00');
```
Comment (andreisamoila74) [inline codewell-fe/src/__tests__/Helpers/Utils.test.js:17]:
```suggestion
    expect(convertToLocalTimezone(utcDate))
      .toStrictEqual('2020-01-01 13:00:00');
```

File: codewell-fe/src/__tests__/Helpers/Utils.test.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  test('Conversion to timezone America/Chicago', () => {
+    moment.tz.guess = function () {
+      return  'America/Chicago';
+    }
+    expect(convertToLocalTimezone(utcDate)).toStrictEqual(
```
Comment (andreisamoila74) [inline codewell-fe/src/__tests__/Helpers/Utils.test.js:24]: same here as above

File: codewell-fe/src/__tests__/Helpers/Utils.test.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      '2020-01-01 06:00:00');
+  });
+
+  test('Conversion to timezone Australia/Darwin', () => {
+    moment.tz.guess = function () {
+      return  'Australia/Darwin';
+    }
+    expect(convertToLocalTimezone(utcDate)).toStrictEqual(
```
Comment (andreisamoila74) [inline codewell-fe/src/__tests__/Helpers/Utils.test.js:32]: same here as above

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

**Review by andreisamoila74 — APPROVED** [review]

**Review by S7012MY — APPROVED** [review]

**Pull request merged**